pennylane_lightning/core/src/simulators/lightning_qubit/measurements/MeasurementsLQubit.hpp

```diff
@@ -100,64 +101,40 @@ class Measurements final
     probs(const std::vector<std::size_t> &wires,
           [[maybe_unused]] const std::vector<std::size_t> &device_wires = {})
         -> std::vector<PrecisionT> {
-        // Determining index that would sort the vector.
-        // This information is needed later.
-        const auto sorted_ind_wires = Pennylane::Util::sorting_indices(wires);
```

Comment:
Seems like sorting is not necessary, why was it added again?

Reply:
It is possible that we changed the logic with time and forgot to update here.